src/pages/Beta/Swap/MyPortfolio/styleds.tsx

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components'
 
 export const PageWrapper = styled(Box)`
   width: 100%;
-  height: 100%;
+  height: 500px;
```

Comment:
what is the reason to change this height here ? I think this task dont related to it.